pkg/estimator/pb/service.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto2";
```

Comment:
Why not use `proto3`?

Reply:
In `types.go`, we use package `corev1`. After generating, we will get the `generated.proto` of `types.go`, which has a dependency relationship with `corev1` [generated proto file](https://github.com/kubernetes/api/blob/master/core/v1/generated.proto). This file use `proto2` so we have no choice to use `proto3`.

Reply:
We will use `k8s.io/code-generator/cmd/go-to-protobuf` to generate proto files.

Reply:
Do you mean the `go-to-protobuf` only supports generating the `proto2` files?

Reply:
I think so, according to [here](https://github.com/kubernetes/code-generator/blob/ff0bf9f438540217af93d6ba0db0a680975e8067/cmd/go-to-protobuf/protobuf/generator.go#L743).